Purchases.xcodeproj/project.pbxproj

```diff
@@ -699,15 +699,16 @@
 			isa = PBXGroup;
 			children = (
 				2CD72940268A820E00BFC976 /* FoundationExtensions */,
-				B3DDB55726854850008CCF23 /* Public */,
```

Comment:
This is a reorganization, alphabetical folders, then files, with the addition of `Networking` folder